tests/unit/models/360-core.js

```diff
@@ -165,6 +165,54 @@ describe("SerSol 360 Core Model >", function() {
     });
   });
 
+  describe("serSol360Core model showJournalCoverImages method >", function() {
+    beforeEach(function() {
+      delete browzine.journalCoverImagesEnabled;
+    });
+
+    afterEach(function() {
+      delete browzine.journalCoverImagesEnabled;
+    });
+
+    it("should show journal cover when configuration property is undefined or null", function() {
+      expect(serSol360Core.showJournalCoverImages()).toEqual(true);
+    });
+
+    it("should show journal cover when configuration property is true", function() {
+      browzine.journalCoverImagesEnabled = true;
+      expect(serSol360Core.showJournalCoverImages()).toEqual(true);
+    });
+
+    it("should not show journal cover when configuration property is false", function() {
+      browzine.journalCoverImagesEnabled = false;
+      expect(serSol360Core.showJournalCoverImages()).toEqual(false);
```

Comment:
Glad we've got some tests here! However, should we add a test that ensures when `serSol360Core.showJournalCoverImages()` is false, the cover image truly does not show up in the record?  Perhaps a test similar to [this one](https://github.com/thirdiron/browzine-discovery-service-adapters/blob/dd8eeb223016c8aeebf952afba98d8ee7154d109/tests/unit/models/360-core.js#L407), except checking for the image to not appear?